shared/source/os_interface/linux/drm_neo.h

```diff
@@ -152,6 +152,12 @@ class Drm : public DriverModel {
     void setDirectSubmissionActive(bool value) { this->directSubmissionActive = value; }
     bool isDirectSubmissionActive() const { return this->directSubmissionActive; }
 
+    void setVmBindSystemAlloc(bool value) { this->vmBindSystemAlloc = value; }
```

Comment:
please remove, this needs to be passed via arguments and not by modyfing global drm state.